### PR TITLE
add DisplayTodos logic

### PR DESCRIPTION
**Pull request by JenniferHIM**

### CODE REVIEW

File: src/components/DisplayTodos.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  const [sort, setSort] = useState('active');
+  return (
+    <div className={styles.displayTodos}>
+      <div className={styles.buttons}>
+        <button onClick={() => setSort('active')}>Active</button>
+        <button onClick={() => setSort('completed')}>Completed</button>
+        <button onClick={() => setSort('all')}>All</button>
+      </div>
+      <ul>
+        {props.todos.length > 0 && sort === 'active'
```
Comment (ZamkovaTetiana) [inline src/components/DisplayTodos.js]: props.todos.length instead of props.todos.length > 0

File: src/components/DisplayTodos.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+          ? props.todos.map((item) => {
+            return (
+              item.completed === false && (
```
Comment (ZamkovaTetiana) [inline src/components/DisplayTodos.js]: !item.completed instead of item.completed === false

File: src/components/DisplayTodos.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+              <TodoItem
+                key={item.id}
+                item={item}
+                removeTodo={props.removeTodo}
+                updateTodo={props.updateTodo}
+                completeTodo={props.completeTodo}
+              />
+            );
+          })
+          : null}
```
Comment (ZamkovaTetiana) [inline src/components/DisplayTodos.js]: rework with switch case for sort

File: src/styles/main.scss [hunk 4]
```diff
@@ -20,21 +21,21 @@ body {
...
-  color: #222;
+  color: $black-color;
   overflow: hidden;
 }
 
 .App {
-  margin-top: 3rem;
+  margin-top: 48px;
   display: flex;
   flex-direction: column;
```
Comment (ZamkovaTetiana) [inline src/styles/main.scss]: use mixin

File: src/styles/main.scss [hunk 5]
```diff
@@ -62,22 +63,68 @@ body {
...
+.displayTodos .buttons {
+  margin-bottom: 32px;
+}
+
+.displayTodos .buttons button {
+  padding: 8px 20px;
+  border-radius: 20px;
+  cursor: pointer;
+  border: none;
+}
```
Comment (ZamkovaTetiana) [inline src/styles/main.scss]: use nesting in styles:
.displayTodos {
  margin-top: 48px;
...
  
  .buttons {
  margin-bottom: 32px;
}

...
}

File: src/App.js [hunk 6]
```diff
@@ -1,12 +1,29 @@
 import React from 'react';
 import styles from './styles/main.scss';
 import Todos from './components/Todos';
+import DisplayTodos from './components/DisplayTodos';
+import { motion } from "framer-motion";
```
Comment (ZamkovaTetiana) [inline src/App.js]: cool :)

Comment (JenniferHIM) [reply src/App.js]: Thanks))

**Review by ZamkovaTetiana — APPROVED** [review]

**Pull request merged**